recipes/onyxgenai/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "onyxgenai" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MetroStar/onyx-genai-sdk/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 0c63da84194a96257e9d051f189054de631b3f2aa9dc672fb2e4310e7794ffe1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+    - setuptools
+  run:
+    - python >=3.10
+    - requests
+    - spacy
+    - PyPDF2
+    - python-pptx
+
+test:
```

Comment:
could you add a `pip check` test?

https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L62-L65